erpnext/hr/doctype/employee/employee.json

```diff
@@ -843,12 +844,18 @@
    "fieldname": "twitter",
    "fieldtype": "Data",
    "label": "Twitter"
+  },
+  {
+   "fieldname": "email_id_of_reports_to",
+   "fieldtype": "Data",
+   "label": "Email ID of Reports to",
+   "options": "Email"
```

Comment:
when the field type is data then there is no need to add options and in options we mention DocType and We dont have any Email Doctype

Reply:
Will update accordingly.

Reply:
- @vishdha We add Email when the fieldtype is Data, so that the data field can run email validations.
- Check `data.js` for reference

Reply:
@hrwX it will validate email in employee doctype there is no need to validate here too. the use case here is it should not change by any user.